cmd/cmd_test.go

```diff
@@ -140,7 +127,12 @@ func TestGetDependenciesList(t *testing.T) {
 		assert.NoError(t, err)
 	}()
 	originSumFileContent, _, err := GetGoSum(gomodPath)
-
+	err = fileutils.MoveFile(filepath.Join(gomodPath, "test.go.txt"), filepath.Join(gomodPath, "test.go"))
```

Comment:
This comment is relevant to the entire function - 
I suggest we avoid moving source files. Yes - these are testdata files, but still, they aer part of the project. Moving these files can cause git changes and other issues. We should instead copy the files into a temp folder, which is outside of the source code directory tree, and perform the test there. We should clean up the copied files using defer at the end.